src/solve.jl

```diff
@@ -28,7 +28,7 @@ function init{uType,tType,isinplace,algType<:Union{AbstractRODEAlgorithm,Abstrac
               qoldinit=1//10^4, fullnormalize=true,
               beta2=beta2_default(alg),
               beta1=beta1_default(alg,beta2),
-              delta=1//6,maxiters = 1e9,
+              delta=map(x->1//6,reltol),maxiters = 1e9,
```

Comment:
Yes, it would be better to make them different types. I don't think vector delta is sensible.